src/modules/Forms/Controls/RangeSlider.js

```diff
@@ -1,38 +1,28 @@
-import React, { useState, useEffect } from 'react';
+import React  from 'react';
```

Comment:
There's an extraneous whitespace between `React` and `from`.

```suggestion
import React from 'react';
```